libsolidity/ast/Types.cpp

```diff
@@ -625,6 +625,7 @@ TypePointer RationalNumberType::binaryOperatorResult(Token::Value _operator, Typ
 	{
 		rational value;
 		bool fractional = isFractional() || other.isFractional();
+		using boost::multiprecision::pow;
```

Comment:
Please don't move that up here, it is only three letters and the distance between declaration and usage is quite high.